src/app/pages/wallets/wallet-details/empty-transaction-list/pages-wallet-details-empty-transaction-list.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
```

Comment:
This should be reusable component so it should be place for example in `/common/components` and called `<app-no-results></app-no-results>`. Besides that according to issue description it should be configurable so it should have `@Input public title: string = 'Ooops!'` which has default value and `@Input() public text!: string` which we always need to provide.